cbmackintosh/intention-timer
# Change error handling colors to yellow/ add line color change

*What does this PR do?*
Changes the color of the errors to yellow as well as the changing the color of the svg image. 
NOTE: new svg image was locally created and saved into assets folder. 

**Clearly and concisely describe the feature.**
- Error handling messages changed color to yellow to stand out more from provided comp. 

**Is this a feature or refactor?**
- refactor

**Problems Encountered & Resulting Solutions**
- NA

**State any problems encountered >Describe your code changes in detail for reviewers.**
- NA

**Where should the reviewer start?**

- Added/ Removed classList lines 102-108
- Added/ Removed classList lines 112-118 
- Added/ Removed classList lines 122-128

**What file(s) and line(s) are where changes are made?**
- Main.js: 102-108 / 112-118 / 122-128
- styles.css: 185-193 / 200 (color change to yellow) 

**How should this be tested?**
- hit START ACTIVITY button without filling in any fields and should see YELLOW error messages and YELLOW images

**What steps need to be taken to confirm functionality?**
- NA

**Steps To Take**
- NA

Any questions and possible steps moving forward.
- NA
